src/InfoBoxes/Content/Type.hpp

```diff
@@ -146,13 +146,20 @@ namespace InfoBoxFactory
     e_ActiveRadio, /* Active Radio Frequency */
 
     e_StandbyRadio, /* Standby Radio Frequency */
+
+    e_TransponderCode, /* Transponder code */
```

Comment:
.... what?

Reply:
Yea, how did that happen?!? I will fix it.